docs/user/vsphere/customization.md

```diff
@@ -9,6 +9,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `password` (required string): The password to use to connect to the vCenter.
 * `datacenter` (required string): The name of the datacenter to use in the vCenter.
 * `defaultDatastore` (required string): The default datastore to use for provisioning volumes.
+* `folder` (optional string): The name of the existing folder where the installer should create VMs. The value should be relative to the datacenter's VM folder. To install VMs in `/example_datacenter/vm/example_folder/example_subfolder` specify the value `example_folder/example_subfolder`. If a value is specified, the folder must exist. If no value is specified, a folder named with the cluster ID will be created in the `datacenter` VM folder.
```

Comment:
can we update the godoc for the field inaddition to this customization document.